### DISCUSSION TITLE
Idea for Curator: Ability to pin comments to a specific location during georef review

### DISCUSSION THREAD

**Discussion by emilyrbowe** [allmaps/allmaps#163] [Ideas for Apps: Editor, Viewer, ...]
@itspangler and I were just talking as we reviewed an intern's multi-plate georeferencing process that it would be great if there were a feature in Curator that would allow someone reviewing to leave "comments" that are pinned to specific sections of the map (i.e. "this mask needs adjusting", "control points for this plate seem off", etc). Ideally, these are things that become "tasks" or something similar for the person georerencing. 

There's definitely more to think about for how this might work, but Ian and I wanted to get it written down before we forgot!

**Comment by bertspaan** [comment]
This is a great idea! I can see 2 ways of implementing this, without turning Curator into a complete discussion platform:

1. A comment is just a tag. Any item in a _List_ can have one or more tags. A tag can be something like `improve_mask`.
2. An item in a _List_ can have a comment. A comment is a string. Users can set this comment, change it, delete it, but that's it: you can't, for example, reply to a comment
